test/Identity.test.js

```diff
@@ -1,9 +1,5 @@
-import {ETH_MESSAGE_AUTH} from "./constants";
-
-const assertEvent = require('./tools/contractEvents').assertEvent;
+const ETH_MESSAGE_AUTH = require('./constants').ETH_MESSAGE_AUTH;
```

Comment:
For improving my own javascript coding style. 
Is this the current recommended way to import a single variable?

Reply:
there are mutiple package standards in js: AMD, commonjs (require, module.export) and ES6(import, export). Node has common js and Es6 is not supported across the board. You have to use something like babel to transpile the code and you can use them combined. We had that to use babel because we had some Es6 modules in openzeppelin but they switched to commonjs and there is no need to have babel and not use just one way of importing stuff, in this case commonjs

Reply:
our import statements were autocomplete stuff from the Webstorm, I did not care that much because we had babel

Reply:
```const ETH_MESSAGE_AUTH = require('./constants').ETH_MESSAGE_AUTH``` 
or 
```const {ETH_MESSAGE_AUTH} = require('./constants'); ```
are the same thing
The second is using object deconstruction(so less efficient) and both are fine for one var. The second one is nicer if you want to import multiple variables from the same package in order not to have less code

Reply:
Thanks for the information. I know that `import` is ES6 and you need the compile it to normal JS.   I didn't know the part with the efficiency :)

Reply:
A good analogy is synonyms in a language. Maybe you prefer a word but the others mean the same thing. Should force people not to use synonyms or should you learn the new word?